explore-analyze/visualize/lens.md

```diff
@@ -408,19 +408,107 @@ In the legend, click the field, then choose one of the following options:
 * **Filter out value** — Applies a filter that removes the field data from the visualization.
 
 
-### Configure the visualization components [configure-the-visualization-components]
+## Customize the visualization display [configure-the-visualization-components]
 
 Each visualization type comes with a set of components that you access from the editor toolbar.
 
 The following component menus are available:
 
-* **Visual options** — Specifies how to display area, line, and bar chart options. For example, you can specify how to display the labels in bar charts.
+* **Appearance** — Specifies how to display area, line, and bar chart options. For example, you can specify how to display the labels in bar charts.
 * **Labels** — Specifies how to display the labels for donut charts, pie charts, and treemaps.
 * **Legend** — Specifies how to display the legend. You can choose to display the legend inside or outside the visualization, truncate the legend values when they’re too long, and [select additional statistics to show](#customize-visualization-legend).
 * **Left axis**, **Bottom axis**, and **Right axis** — Specify how you want to display the chart axes. For example, add axis labels and change the orientation and bounds.
 
+### Visualization appearance options [customize-visualization-appearance]
 
-#### Customize a visualization legend [customize-visualization-legend]
+When creating or editing a visualization, you can customize several appearance options. To do that, look for the {icon}`brush` icon.
+
+These options can vary depending on the type of chart.
+
+#### Area, Bar, and Line charts
+
+**Area fill opacity**
+:   For **Area** charts. Opacity of the area fill. Defaults to `0.3`.
+
+**Bar orientation**
```

Comment:
even some screenshot of each configuration results could be nice, not needs to be very large, just 250x150 pixel is enough to show most of the configurations

Reply:
As is, this will still make the page a lot more cluttered. I'd also leave this to when we tackle https://github.com/elastic/docs-content/issues/1699 and have a page/section dedicated to each chart type.